src/install/deploySteps.js

```diff
@@ -175,6 +174,23 @@ module.exports = (apps, mode, deployDoc) => {
     }));
   };
 
+  const removeOldVersion = changedApps => {
```

Comment:
This code shouldn't be in this file as it's not about deploying anymore. It should be moved to where postDeployCleanup is (if you think that chunk of code is getting too complicated feel free to spin it out into its own file).

FWIW this file is kind of a legacy bucket of code: you'll notice most of the code is inside the export that sets "global" variables. Ideally I'd like us to slowly refactor this code so we don't need to do that anymore.

Reply:
We also shouldn't have to pass in changedApps.

We know what apps exist based on the `deployDoc.node_modules` parameter. We can just check those for the old symlink I think and do that instead.